src/api/test/functional/attributes_test.rb

```diff
@@ -131,6 +126,10 @@ def test_create_type
     assert_response :success
     get "/attribute/TEST/Dummy/_meta"
     assert_response :success
+    login_Iggy
+    delete "/attribute/TEST/Dummy/_meta"
+    assert_response 403
+    login_adrian
```

Comment:
Since we are in the process of moving these tests over to rspec, why not write this test in rspec?